packages/omd/omd.2.0.0.alpha

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+maintainer: "Philippe Wang <philippe.wang@gmail.com>"
+authors: [ "Philippe Wang <philippe.wang@gmail.com>" ]
+license: "ISC"
+homepage: "https://github.com/ocaml/omd"
+dev-repo: "git+https://github.com/ocaml/omd.git"
+bug-reports: "https://github.com/ocaml/omd/issues"
+tags: [ "org:ocamllabs" "org:mirage"  ]
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+build-test: [
```

Comment:
This has been renamed to `run-test` in opam2: https://opam.ocaml.org/doc/Manual.html#opamfield-run-test

Reply:
OK